lib/iris/tests/benchmarking/regridding.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+
+"""
+This file currently exists as a simple demonstration of Airspeed Velocity
+performance testing. it is expected to be expanded/replaced in future.
+
+"""
+
+# import iris tests first so that some things can be initialised before
+# importing anything else.
+from iris import tests
+
+import iris
+
+from iris import analysis
```

Comment:
Instead, perhaps use:
```python
from iris.analysis import AreaWeighted
```

Reply:
👍  I need to break the habit of unecessary future-proofing